### ISSUE TITLE
versions might be str or int

### ISSUE THREAD

**Issue by FlorianLudwig** [andriykohut/gsem#4]
```
 $ gsem outdated
Traceback (most recent call last):
  File "/home/f.ludwig/.local/bin/gsem", line 11, in <module>
    load_entry_point('gsem==0.1.2', 'console_scripts', 'gsem')()
  File "/home/f.ludwig/.local/lib/python3.6/site-packages/gsem/cli.py", line 105, in main
    result = list_cmd_map[args.cmd]()
  File "/home/f.ludwig/.local/lib/python3.6/site-packages/gsem/extension.py", line 109, in outdated
    return [e for e in self.enabled() if e.outdated()]
  File "/home/f.ludwig/.local/lib/python3.6/site-packages/gsem/extension.py", line 109, in <listcomp>
    return [e for e in self.enabled() if e.outdated()]
  File "/home/f.ludwig/.local/lib/python3.6/site-packages/gsem/extension.py", line 52, in outdated
    return self.remote_meta['version'] > self.meta['version']
TypeError: '>' not supported between instances of 'int' and 'str'
```

**Comment by andriykohut** [comment]
Hey @FlorianLudwig! Thanks for reporting, could you please paste a list of installed extensions? (`ls ~/.local/share/gnome-shell/extensions/`). Trying to narrow down the culprit.

**Comment by FlorianLudwig** [comment]
Sure thing:
```
alternate-tab@gnome-shell-extensions.gcampax.github.com
apps-menu@gnome-shell-extensions.gcampax.github.com
cariboublocker@git.keringar.xyz
gsconnect@andyholmes.github.io
impatience@gfxmonk.net
launch-new-instance@gnome-shell-extensions.gcampax.github.com
TopIcons@phocean.net
user-theme@gnome-shell-extensions.gcampax.github.com
window-list@gnome-shell-extensions.gcampax.github.com
```

**Comment by andriykohut** [comment]
This should be fixed in `0.1.3`.
